Tasks/Maven/maventask.ts

```diff
@@ -203,13 +208,22 @@ function createMavenSQRunner(sqHostUrl, sqHostUsername, sqHostPassword, sqDbUrl?
     return mvnsq;
 }
 
+// check for and, if appropriate, add PMD analysis goals
+function applyPmdGoals(mvnRun: trm.ToolRunner):void {
+    if (tl.getInput('pmdAnalysisEnabled', true) != 'true') {
+        console.log('PMD analysis is not enabled');
```

Comment:
I'd argue this is not a message that is important enough so it could be a debug message. Then again, the Maven task outputs hundreds of lines of logs, so it doesn't really matter.